appengine/standard-datastore/Gemfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2015 Google, Inc.
```

Comment:
Please use

```
# Copyright 2020 Google LLC
...
```

for all copyright notices.

Reply:
Fixed